javadoc/pom.xml

```diff
@@ -332,11 +332,6 @@
       <artifactId>jetty-start</artifactId>
       <scope>provided</scope>
     </dependency>
-    <!--<dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-unixsocket-client</artifactId>
-      <scope>provided</scope>
-    </dependency>-->
     <!--<dependency>
```

Comment:
I'd get rid of these other commented out dependency, they look leftovers from the 11 module names.